pkgs/tools/admin/gixy/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, fetchFromGitHub, python, pythonPackages
+, python27Packages, python35Packages, python36Packages }:
+let
+  pyPkgs = (if python.pythonVersion=="3.5"
```

Comment:
This needs to disappear. You can just use `python.pkgs` instead of `pythonPackages`.